boards/infineon/cyw920829m2evk_02/cyw920829m2evk_02.dts

```diff
@@ -72,34 +72,45 @@ uart2: &scb2 {
 };
 
 / {
-	flash0: flash@60000000 {
-		compatible = "soc-nv-flash";
-		reg = <0x60000000 DT_SIZE_K(512)>;
-		write-block-size = <16>;
-		erase-block-size = <256>;
-
-		partitions {
-			compatible = "fixed-partitions";
+	qspi_flash: qspi_flash@40890000 {
+		compatible = "infineon,cat1-qspi-flash";
+		reg = <0x40890000 0x30000>;
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		flash0: flash@60000000 {
+			compatible = "soc-nv-flash";
+			reg = <0x60000000 DT_SIZE_K(512)>;
+			write-block-size = <1>;
+			erase-block-size = <DT_SIZE_K(4)>;
 			#address-cells = <1>;
 			#size-cells = <1>;
 
-			toc2_region: flash@60000000 {
+			toc2_region: toc2_region@60000000 {
 				compatible = "zephyr,memory-region", "soc-nv-flash";
 				zephyr,memory-region = "APP_HEADER_FLASH";
 				reg = <0x60000000 0x50>;
 			};
-			bootstrap_region: flash@60000050 {
+			bootstrap_region: bootstrap_region@60000050 {
 				compatible = "zephyr,memory-region", "soc-nv-flash";
 				zephyr,memory-region = "BOOTSTRAP_FLASH";
 				reg = <0x60000050 0x2550>;
 			};
-			app_region: flash@60002600 {
+
+			app_region: app_region@60002600 {
 				compatible = "soc-nv-flash";
 				reg = <0x60002600 0x5da00>;
```

Comment:
can you use DT_SIZE_K(374)>; here as well

Reply:
already updated @ https://github.com/zephyrproject-rtos/zephyr/pull/74174